[23.2] Initial release notes  for 23.2

dannon opened the pull request:
Ref #16742

To do:
- [x] insert proper links to videos in user release notes
- [x] add proper alt text to images, video

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

dannon committed:
Release note scaffold for 23.2

dannon committed:
Link announce for 23.2

mvdbeek commented on doc/source/releases/23.2.rst:
```diff
@@ -0,0 +1,950 @@
+
+.. feature
+
+* Make form repeat blocks reordarable
```
Those are PRs that already went into 23.1 ...

dannon replied:
Which ones?  They're all milestoned 23.2, and checking at least the first few (with `git branch -a --contains <blah>`) I don't see any issues.

dannon committed:
Add updated primary announce content.

This needs to be fixed in galaxy-release-util; by default updating this
is skipped, but it'll hold the previous announcement placeholder.
Should detect placeholder and overwrite if it's there.

dannon committed:
Adjust naming to mid/late?

dannon committed:
Drop New Visualizations section -- there are no new ones, only fixes

dannon committed:
Add config changes

jdavcs committed:
Remove release testing section

There was no formal release testing for this release

jdavcs committed:
Add link to admin highlights

jdavcs committed:
Use semantic headings for highlights

TODO: This needs to be updated in the script

jdavcs committed:
Cleanup PR lists

Reformat
Fix wording
Drop irrelevant PRs from user-facing notes

jdavcs committed:
Add user release notes

TODO:
1. Add correct links to videos
2. Add image + video alt descriptions

jdavcs committed:
Add release notes highlights + admin notes to main announcement

jdavcs committed:
Add videos to release notes

jdavcs committed:
Add alt text to images in release notes

jdavcs reviewed (changes requested):
video links need editing; maybe rearrange the figures for workflow comments for a better flow.

jdavcs committed:
Fix video embed links: Do not play videos from the same playlist

jdavcs committed:
Improve layout, remove (almost) redundant video
